script/functions.sh

```diff
@@ -12,10 +12,8 @@ SPECS_HAVE_RUN_FILE=specs.out
 MAINTENANCE_BRANCH=`cat maintenance-branch`
 
 # Don't allow rubygems to pollute what's loaded. Also, things boot faster
-# without the extra load time of rubygems. Only works on MRI Ruby 1.9+
-if is_mri_192_plus; then
-  export RUBYOPT="--disable=gem"
-fi
+# without the extra load time of rubygems.
+export RUBYOPT="--disable=gem"
```

Comment:
Sure, that's part of the iterative plan.

> for rspec-dev:
>   remove unused predicate_functions

I'm experimenting here to figure out what can be stripped off, so we can find what needs to remain because it's still used in some of the repos and what can be removed in `rspec-dev`. I'll extract this and will send a follow-up PR to `rspec-dev` and to other repos against `4-0-dev`.